lib/errors.ts

```diff
@@ -61,6 +61,8 @@ export const GitErrorRegexes = {
   'The requested URL returned error: 403': GitError.HTTPSAuthenticationFailed,
   'fatal: The remote end hung up unexpectedly': GitError.RemoteDisconnection,
   "fatal: unable to access '(.+)': Failed to connect to (.+): Host is down": GitError.HostDown,
+  "Cloning into '(.+)'...\nfatal: unable to access '(.+)': Could not resolve host: (.+)\n":
```

Comment:
For the most part these regexes can be relaxed to ignore control characters at the ends of strings, unless we start exploring returning values included in the groups found in each regex match:

```suggestion
  "Cloning into '(.+)'...\nfatal: unable to access '(.+)': Could not resolve host: (.+)":
```